.replit

```diff
@@ -1,2 +1,2 @@
-language = "python3"
```

Comment:
One thing, keep the language="python3" pls

Reply:
That's not needed anymore.

Reply:
Oh ok 👍